src/renderer/components/ImageCropper/index.tsx

```diff
@@ -0,0 +1,284 @@
+      return
+    }
+
+    const imgW = fullImage.current.clientWidth
+    const imgH = fullImage.current.clientHeight
+    const ctxW = container.current.clientWidth
+    const ctxH = container.current.clientHeight
+
+    // cut zoom on width
+    zoom.current = Math.min(
+      imgW / targetWidth,
+      Math.max(targetWidth / imgW, zoom.current)
+    )
+
+    // cut zoom on height
+    zoom.current = Math.min(
+      imgH / targetHeight,
+      Math.max(targetHeight / imgH, zoom.current)
+    )
+
+    // x = Math.max(Math.min((imgW - targetWidth) / 2, x), (targetWidth - imgW) / 2)
+    // y = Math.max(Math.min((imgH - targetHeight) / 2, y), (targetHeight - imgH) / 2)
+
+    placeSelector(imgW / 2 + x, imgH / 2 + y)
+
+    cutImage.current.style.left = fullImage.current.style.left =
+      Math.round((ctxW - imgW) / 2 - x * zoom.current) + 'px'
+    cutImage.current.style.top = fullImage.current.style.top =
+      Math.round((ctxH - imgH) / 2 - y * zoom.current) + 'px'
+
+    return [x, y]
+  }
+
+  useLayoutEffect(() => {
+    const onMouseDown = (ev: MouseEvent) => {
+      ev.preventDefault()
+
+      dragging.current = true
+
+      startX.current = ev.clientX
+      startY.current = ev.clientY
+    }
+
+    const onMouseUp = (ev: MouseEvent) => {
+      ev.preventDefault()
+
+      dragging.current = false
+
+      posX.current +=
+        (ev.clientX - startX.current) * Math.min(1 / zoom.current, 1)
+      posY.current +=
+        (ev.clientY - startY.current) * Math.min(1 / zoom.current, 1)
+      moveImages(posX.current, posY.current)
+    }
+
+    const onMouseMove = (ev: MouseEvent) => {
+      if (!dragging.current) {
+        return
+      }
+      // we never go faster with zoom above 1, but we speed up zoomed-out (below 1) by inverting the zoom value
+      const dX = (ev.clientX - startX.current) * Math.min(1 / zoom.current, 1)
+      const dY = (ev.clientY - startY.current) * Math.min(1 / zoom.current, 1)
+
+      moveImages(posX.current + dX, posY.current + dY)
+    }
+
+    const onZoom = (ev: WheelEvent) => {
+      // negate delta so when we wheel forward we zoom in
+      zoom.current *= Math.exp((ev.deltaY > 1 ? -1 : 1) * 0.2)
+
+      moveImages(posX.current, posY.current)
+    }
+
+    document.addEventListener('mousemove', onMouseMove)
+    document.addEventListener('mousedown', onMouseDown)
+    document.addEventListener('mouseup', onMouseUp)
+    document.addEventListener('wheel', onZoom)
+
+    return () => {
+      document.removeEventListener('mousemove', onMouseMove)
+      document.removeEventListener('mousedown', onMouseDown)
+      document.removeEventListener('mouseup', onMouseUp)
+      document.removeEventListener('wheel', onZoom)
+    }
+  })
+  return (
+    <Dialog
+      className={styles.imageCropperDialog}
+      canEscapeKeyClose
+      onClose={onClose}
+      canOutsideClickClose={false}
+    >
+      <DialogHeader></DialogHeader>
```

Comment:
I'd give it a title, not for every user it'll be clear what I can "do" in this dialog, especially when it will be always shown